CMakeLists.txt

```diff
@@ -16,15 +16,30 @@ endif(NOT ${INSTALL_FHS})
 # but CMake does the extra quoting for us.
 #add_definitions(-DDATAPATH="/usr/share/games/gish")
 
-find_package(SDL REQUIRED)
-find_package(OpenAL REQUIRED)
-find_package(OpenGL REQUIRED)
-find_package(Vorbis REQUIRED)
-find_package(PNG REQUIRED)
-find_library(M_LIBRARIES m)
+if(MSVC)
+    find_package(Ogg CONFIG REQUIRED)
+    find_package(Vorbis CONFIG REQUIRED)
+    find_package(libpng CONFIG REQUIRED)
+    find_package(SDL2 CONFIG REQUIRED)
+    find_package(OpenAL CONFIG REQUIRED)
+    find_package(OpenGL REQUIRED)
+
+else()
+    # Find SDL2 with pkg-config
+    include(FindPkgConfig)
+    pkg_search_module(SDL2 REQUIRED sdl2)
```

Comment:
Using `find_package(SDL2 REQUIRED)` also seems to work on my Windows/Linux machines.

Reply:
I tested this on Ubuntu 22.04 and this seems to work. I'll change it.